### PR TITLE
Add support for multiple headers

### PR DESCRIPTION
**Pull request by LachlanMcCulloch**

### CODE REVIEW

File: .swiftpm/xcode/package.xcworkspace/contents.xcworkspacedata [hunk 1]
```diff
@@ -0,0 +1,7 @@
+<?xml version="1.0" encoding="UTF-8"?>
+<Workspace
+   version = "1.0">
+   <FileRef
+      location = "self:">
+   </FileRef>
+</Workspace>
```
Comment (designatednerd) [inline .swiftpm/xcode/package.xcworkspace/contents.xcworkspacedata:7]: I don't believe this needs to be committed

File: Tests/ApolloCodegenTests/ApolloSchemaTests.swift [hunk 2]
```diff
@@ -54,7 +57,8 @@ class ApolloSchemaTests: XCTestCase {
         "--endpoint=http://localhost:8080/graphql",
         "--key=\(apiKey)",
         "'\(expectedOutputURL.path)'",
-        "--header='\(header)'"
+        "--header='\(headers.first!)'",
+        "--header='\(headers.last!)'"
```
Comment (designatednerd) [inline Tests/ApolloCodegenTests/ApolloSchemaTests.swift:61]: I'd prefer these not have force-unwraps - maybe declare it as two vars and then shove those into the header array. However, this is something I can fix post-merge if you prefer.

File: Tests/ApolloTests/VersionNumberTests.swift [hunk 3]
```diff
@@ -4,6 +4,6 @@ import XCTest
 class VersionNumberTests: XCTestCase {
   func testVersionNumberExists() {
     // It would be the first 2 digits of version number like `0.19`.
-    XCTAssertGreaterThanOrEqual(ApolloVersionNumber, 0)
+//    XCTAssertGreaterThanOrEqual(ApolloVersionNumber, 0)
```
Comment (designatednerd) [inline Tests/ApolloTests/VersionNumberTests.swift:7]: This should not be committed - I know it doesn't run right if you're not using the schemes, but I'd still prefer to run it with the schemes.

File: .swiftpm/xcode/package.xcworkspace/contents.xcworkspacedata (hunk 1 above)
Comment (LachlanMcCulloch) [reply .swiftpm/xcode/package.xcworkspace/contents.xcworkspacedata]: Have removed - I'm unfamiliar with SPM 😅

File: Tests/ApolloCodegenTests/ApolloSchemaTests.swift (hunk 2 above)
Comment (LachlanMcCulloch) [reply Tests/ApolloCodegenTests/ApolloSchemaTests.swift]: Too easy - have created two separate variables labelled `firstHeader` and `lastHeader` for clarity and to remove the force unwraps

File: Tests/ApolloTests/VersionNumberTests.swift (hunk 3 above)
Comment (LachlanMcCulloch) [reply Tests/ApolloTests/VersionNumberTests.swift]: Yep, this was my bad, I meant to uncomment and should be back to normal now

**Pull request merged**